### PR TITLE
Development

### PR DESCRIPTION
**Pull request by codewarsfx**
improve result card ui making important weather detail pop more

### CODE REVIEW

**Comment by awilkescom** [comment]
@LiubovRukhlina can you please review and merge?

File: example.env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_APIKEY = e83d65c81273c20ea343f3e63c652f10
```
Comment (LiubovRukhlina) [inline example.env:1]: remove the key please

File: src/App.css [hunk 2]
```diff
@@ -68,3 +69,27 @@ input {
 .card {
   display: inline-block;
 }
+.result_title{
+   font-size: 32px;
+   font-family: 'Open Sans', sans-serif;
```
Comment (LiubovRukhlina) [inline src/App.css:74]: The project has already 4 different fonts. This one is nice, but I don't think it should be defined on the line level. What do you think? maybe redefine for the whole project

File: src/App.css [hunk 3]
```diff
@@ -68,3 +69,27 @@ input {
...
+   font-weight: 700;
+   margin: 0;
+   display: flex;
+   align-items: center;
+   justify-content: center;
+}
+
+.result_description{
+  font-family: 'Poppins', sans-serif;
```
Comment (LiubovRukhlina) [inline src/App.css:83]: again a new font. check the comment above

File: src/App.css [hunk 4]
```diff
@@ -68,3 +69,27 @@ input {
...
 }
+.result_title{
+   font-size: 32px;
+   font-family: 'Open Sans', sans-serif;
+   font-weight: 700;
+   margin: 0;
+   display: flex;
+   align-items: center;
+   justify-content: center;
+}
```
Comment (LiubovRukhlina) [inline src/App.css:80]: I think the title should have the same color as other lines in result window #444

File: src/Components/Forecast/BookmarkIcon/bookmarkIconComponent.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import './bookmarkIconStyles.css'
+import {BsBookmarkHeartFill} from 'react-icons/bs'
+import { useState } from 'react'
+
+const BookmarkIcon= ()=>{
+const [isClicked,setIsClicked] =useState(false)
+
+return(
+     <BsBookmarkHeartFill className="bookmark-icon" style={{color:isClicked?"rgb(17, 64, 110)":"#333"}} onClick={()=>setIsClicked(!isClicked)}/>
```
Comment (LiubovRukhlina) [inline src/Components/Forecast/BookmarkIcon/bookmarkIconComponent.jsx:9]: icon doesn't change on click

File: example.env (hunk 1 above)
Comment (codewarsfx) [reply example.env:1]: oh sure thanks for pointing that out

File: src/App.css (hunk 2 above)
Comment (codewarsfx) [reply src/App.css:74]: cool suggestion... I was looking at open sans for the h3 title and a more jolly one like poppins for the body (Line level)... Ill make them global like you said. thanks

File: src/App.css (hunk 4 above)
Comment (codewarsfx) [reply src/App.css:80]: I'm trying to create a distinction for information  hierarchy using font weight and color.

File: src/Components/Forecast/BookmarkIcon/bookmarkIconComponent.jsx (hunk 5 above)
Comment (codewarsfx) [reply src/Components/Forecast/BookmarkIcon/bookmarkIconComponent.jsx:9]: I'll take a look at it ...for now its supposed to change its color when clicked

**Comment by awilkescom** [comment]
Hey - seems my merge caused an issue in App.js. @codewarsfx I'll leave it to you to review the feedback and make another update :)

**Pull request closed without merging**